.github/workflows/main.yml

```diff
@@ -9,10 +9,23 @@ on:
 jobs:
   unit-tests:
     name: Unit tests
+    # Workaround https://github.com/nektos/act/issues/1875
     uses: ./.github/workflows/unit_tests.yml
     with:
       linux_5_9_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -strict-concurrency=complete"
       linux_5_10_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -strict-concurrency=complete"
       linux_6_0_arguments_override: "--explicit-target-dependency-import-check error"
       linux_nightly_6_0_arguments_override: "--explicit-target-dependency-import-check error"
       linux_nightly_main_arguments_override: "--explicit-target-dependency-import-check error"
+
+  benchmarks:
+    name: Benchmarks
+    # Workaround https://github.com/nektos/act/issues/1875
+    uses: ./.github/workflows/benchmarks.yml@main
+    with:
+      benchmark_package_path: "Benchmarks"
+
+  cxx-interop:
+    name: Cxx interop
+    # Workaround https://github.com/nektos/act/issues/1875
```

Comment:
```suggestion
```

This comment is not relevant here, it's to do with chained workflow imports, it's copy-pasta.